### PR TITLE
add more subreddits and add sidebar styling

### PR DESCRIPTION
**Pull request by SuperStormer**
I have not tested on all of the newly added sites yet; only r/anime, r/manga, r/awwnime, r/animenocontext, r/madokamagica so far.

### CODE REVIEW

File: src/contentScript.js
Comment (eritbh) [inline src/contentScript.js:54]: my ESLint config is bad and doesn't handle autofixes for stuff like this well, sorry, can you update this condition to look like:

```js
if ([
    'anime',
    // ...
    'chibi',
].includes(...)) {
    // ...
}
```

File: src/contentScript.js
Comment (eritbh) [inline src/contentScript.js:91]: For my own knowledge, was this motivated by something specific? Does only setting `backgroundColor` cause issues with leftover background images or something?

**Review by eritbh — CHANGES_REQUESTED** [review]
Hey, thanks for the contribution, this looks awesome! I left a couple notes.

I think in the future I'd like to split out the subreddit-specific selector logic into a separate JS or JSON file that maps subreddits to their selectors, but if you don't feel like implementing that I'm also happy to accept this as it is and let that be a project for me later on. Let me know what you think!

File: src/contentScript.js
Comment (SuperStormer) [reply src/contentScript.js:91]: Yeah, r/madokamagica sets a background image on <body>.

File: src/manifest.json
Comment (SuperStormer) [reply src/manifest.json:22]: Oops, forgot about that subdomain, I'll add it back. This change was to prevent the extension from running on old.reddit.com, plus all of the 2 letter language code subdomains (eg. np.reddit.com).

Comment (eritbh) [reply src/manifest.json:22]: That makes sense. I think it should be okay to just specify www and new here instead of the wildcard; I didn't initially realize that it's not possible to get new Reddit on the language subdomains, otherwise I probably would've just done it that way before.

File: src/contentScript.js
Comment (SuperStormer) [reply src/contentScript.js:54]: Hmmm, ESLint complains with https://eslint.org/docs/rules/array-bracket-newline, should I modify the eslint config or just leave the code as is?

**Comment by SuperStormer** [comment]
As for the JSON file for the selector logic, I think we can leave that to be implemented in the future.

File: src/contentScript.js
Comment (eritbh) [reply src/contentScript.js:54]: hmmmmm not worth it since I don't even have CI set up on this repo, let's just leave it as it is and whenever I get around to updating my personal config I'll fix it

**Review by eritbh — APPROVED** [review]
Tested locally, looks good to me! Some subreddits I tested with will need some additional style resets to get rid of pseudo-elements on `html` and `body`, but I'll deal with those after merging. Thanks for the contribution!

**Pull request merged**
